### ISSUE TITLE
yolo_test forward pass

### ISSUE THREAD

**Issue by mohammad-py** [allanzelener/YAD2K#34]
Hi @allanzelener 
Could you please tell me, did you disable batchnorm layer in forward pass when you want to test an input image?
I have got one problem in transfered pytorch model. When I don't disable the `batchnorm` layer, the networks work well but when i disable it, I face with overflow issue in `yolo_head' for sigmoid function.

**Comment by allanzelener** [comment]
The behavior of batch norm is handled by Keras. I don't change it explicitly but it does have different behavior between train and test time.
